src/meta/api/src/schema_api_impl.rs

```diff
@@ -3230,6 +3238,18 @@ fn build_upsert_table_copied_file_info_conditions(
     Ok((condition, if_then))
 }
 
+fn build_upsert_table_deduplicated_label(deduplicated_label: String) -> TxnOp {
+    let expire_at = Some(SeqV::<()>::now_ms() / 1000 + 24 * 60 * 60 * 1000);
+    TxnOp {
+        request: Some(Request::Put(TxnPutRequest {
```

Comment:
1000 days is a little bit long, so there should be a mechanism to clean them up.